modules/quarto-basics/index.qmd

```diff
@@ -0,0 +1,223 @@
+
+![](images/chunk_options.png){width="550" fig-align="center"}
+
+
+Look at the image of the code chunk above, you can see that the chunk has the following options:
+
+`fig-cap`: This option allows you to add a caption to the figure.
+
+`fig-width`: This option specifies the width of the figure.
+
+`echo`: This option controls whether the code in the chunk is displayed in the output.
+
+These are only a couple examples of options for chunk options, and there are many more available. You can find a full list of chunk options in [section 28.5.2](https://r4ds.hadley.nz/quarto) as well as further discussion in Communicating with Quarto. 
+
+
+::: callout-tip
+## Global Options
+To set global options for all code chunks, add them in the YAML header under `execute:`. Indent the options you want to apply by pressing tab on the next line.
+:::
+
+# Markdown text
+
+In a Quarto document, you can write text using Markdown syntax. Markdown is a lightweight markup language that allows you to format text using simple and intuitive syntax. You can use Markdown to create headings, lists, links, images, and more.
+
+## Headings
+
+Headings are created using the `#` symbol followed by the heading text. The number of `#` symbols indicates the level of the heading. For example, `#` creates a first-level heading, `##` creates a second-level heading, and so on.
+
+## Text Formatting
+Just like in any other word processors, you can also format your text. Here are some examples:
+
+- **Bold**: `**bold**` or `__bold__`
+- *Italic*: `*italic*` or `_italic_`
+- ~~Strikethrough~~: `~~strikethrough~~`
+- `Code`: `` `code` ``
+
+## Lists
+
+Lists can be created using `*`. You can also create nested lists by indenting the items with spaces.
+
+::: callout-tip
+## Checklist
+You can also create checklists in Quarto! This can be done by using `- [ ]` for an empty box and `- [x]` for a checked box.
+:::
+
+
+# Rendering Your Document
+
+Once you have written your document, you can render it to see the output. To render your document you can:
+
+1. click the `Render` button in the toolbar
+2. use the keyboard shortcut `Ctrl + Shift + K` (Windows) or `Cmd + Shift + K` (Mac)
+3. run the `quarto::render()` function in the console
+
+![](images/console_render.png){width="700" fig-align="center"}
+
+4. In the Terminal via `quarto render`:
+
+![](images/terminal_render.png){width="700" fig-align="center"}
+
+When you render your document, Quarto will execute the code chunks and generate the output. The output will be displayed in the Viewer pane in RStudio, and you can also view the output in a web browser by clicking cog icon next to `Render` and ticking the `Preview in window` box.
+
+
+## Test your knowledge
+
+** potentially a multiple choice question here about different common chunk options and what they do ??**
+
+** interactive component to edit a basic yaml / include things in yaml -- pain point: warnings, messages = false, toc **
```

Comment:
This could be tricky, I'll see what can be done.
It probably needs to be separate from webr for this, or at least with an webr option that prevents execution of code?